v1/lib/gamefield.php

```diff
@@ -6,21 +6,44 @@
  * Date: 15.01.2016
  * Time: 12:02
  */
+require_once "cell.php";
+
 class GameField
 {
     /** @var  Cell[] */
     private $cells;
+    private $width;
+    private $height;
 
     /**
      * GameField constructor.
      * @param $_cells cell[] Array which contains instances of cell.
      */
-    public function __construct($_cells)
+    public function __construct($_width, $_height)
     {
-        $this->cells = $_cells;
+        $this->width = $_width;
+        $this->height = $_height;
+        $this->cells = array();
+        $this->createCells();
     }
 
     /**
+     * Private function which creates the array of cells which are the gamefield.
+     */
+    private function createCells()
+    {
+        // Make columns
+        for($i=0; $i<$this->height; $i++)
+        {
```

Comment:
Didn't you read the style guide? A for is followed by a space:
for ()

And:
change documenation:
for ()
{ // create columns